core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -138,6 +142,45 @@ oppia.factory('SimpleEditorManagerService', [
         data.questionList.addQuestion(QuestionObjectFactory.create(
           lastStateName, stateData.interaction, ''));
       },
+      changeQuestion: function(type, index) {
+        var currentStateName = (
+          index === 0 ? 'Introduction' : 'Question ' + index);
+        var questionCount = data.questionList._questions.length;
+        var answerGroupsLength = (
+          data.questionList._questions[index]._answerGroups.length);
+        var nextStateName = 'Question ' + (index + 1);
+        var currentInteractionId = (
+          data.questionList._questions[index]._interactionId);
+
+        // Update Question Type If interactionId is not same.
+        if(type !== currentInteractionId) {
+          var newAnswerGroups = [];
+
+          switch (type) {
+            case 'MultipleChoiceInput':
+              SimpleEditorShimService.saveInteractionId(
+                currentStateName, DEFAULT_INTERACTION.ID);
+              SimpleEditorShimService.saveCustomizationArgs(
+                currentStateName, DEFAULT_INTERACTION.CUSTOMIZATION_ARGS);
+              newAnswerGroups.push(AnswerGroupObjectFactory.createNew([
+                RuleObjectFactory.createNew('Equals', {
```

Comment:
Btw, I wonder if we can refactor this so that you don't even have a switch statement. Just look up an explicit constant based on the interaction type, and maybe do minor modifications to them (e.g. for nextStateName). That would be cleaner and would generalize well.

E.g. you could have:
```
DEFAULT_INTERACTION_PROPERTIES: {
  MultipleChoiceInput: {
    id: ...,
    customizationArgs: ...
    answerGroupsTemplate: ...
  }
};
```

and read off DEFAULT_INTERACTION_PROPERTIES[newQuestionType].id, etc.